src/utils/ltfsck.c

```diff
@@ -96,19 +98,19 @@ enum {
 };
 
 struct other_check_opts {
-	struct config_file *config; /**< Configurate data read from the global LTFS config file. */
-	char *devname;              /**< Device to format */
-	char *backend_path;         /**< Path to tape backend shared library */
-	char *kmi_backend_name;     /**< Name or path to the key manager interface backend library */
+	struct config_file* config; /**< Configurate data read from the global LTFS config file. */
+	char* devname;              /**< Device to format */
```

Comment:
Please do not use this kind of declaration and revert to the original.

`char*` is not a type definition in C. So `*` shall be placed just before variable name.

For example, following declaration is confusable.

```C
char* a, b; // a is (char *) and b is (char).
```